exasol_sphinx_github_pages_generator/deploy_github_pages.py

```diff
@@ -96,14 +93,14 @@ def build_and_copy_documentation(build_dir, worktree, source_branch, source_dir,
     output_dir.mkdir(parents=True)
     print(f"Copying HTML output {build_dir} to the output directory {output_dir}")
     for obj in os.listdir(build_dir):
-        shutil.move(build_dir + "/" + str(obj), output_dir)
-    # TODO test correctness of : find "build_dir" -mindepth 1 -maxdepth 1 -exec mv -t "$OUTPUT_DIR" -- {} +
+        if ".doctree" not in str(obj):
+            shutil.move(build_dir + "/" + str(obj), output_dir)
     print(f"Content of output directory {output_dir}")
     open(f"{worktree}/.nojekyll", "w").close()
     run(["ls", "-la", output_dir], check=True)
     return output_dir
 
-#todo remove debug output
```

Comment:
So we need to keep all the logs?

Reply:
This was just a duplicate todo. I will remove the logs and also add the typing and proper comments in the documentation ticket.